cached_data/cached_data.md

```diff
@@ -0,0 +1,3 @@
+# About
+This folder contains sample cached data that can be used to more efficiency run scripts score_generated_collection or
+the resolver to generate OpenRarity and/or external rarity ranks.
```

Comment:
Looks like the cached data made its way into the commit (see below). Agree with Taylor that it might be easier to just ignore a `.cache` folder.

Reply:
I purposefully added it so people have an example of what it looks like - I can do taylor's suggestion if we think we don't want an example?